src/components/UI/PhotoBox.js

```diff
@@ -27,7 +30,12 @@ const PhotoBox = props => {
       data-testid="photo-box"
     >
       <Link to={linkPath}>
-        <img alt={label} src={imgSrc} data-testid="img-photo-box" />
+        <img
+          alt={label}
+          src={imgSrc}
+          data-testid="img-photo-box"
+          onError={loadPlaceholderImage}
```

Comment:
This is not a valid attribute on an HTML element, `onError`, and I don't think a React convention.   Can we implement this logic without using that prop?   Unless I'm missing something...which may be the case.